src/shared/specifier.c

```diff
@@ -175,6 +175,19 @@ int specifier_uuid(char specifier, const void *data, const char *root, const voi
         return 0;
 }
 
+int specifier_uint64(char specifier, const void *data, const char *root, const void *usedata, char **ret) {
+        const uint64_t *n = ASSERT_PTR(data);
+        char *s;
+        int r;
+
+        r = asprintf(&s, "%" PRIu64, *n);
+        if (r < 0)
+                return -errno;
```

Comment:
might as well drop r, and just do:

```
if (asprintf(…) < 0)
        return -ENOMEM;
```

also, asprintf() doesn't document that it sets errno. Hence just hardcode this to ENOMEM?